src/index.js

```diff
@@ -170,7 +170,11 @@ class MiniCssExtractPlugin {
                   renderedModules,
                   compilation.runtimeTemplate.requestShortener
                 ),
-              filenameTemplate: this.options.filename,
+              filenameTemplate: this.getFilename(
+                chunk,
+                this.options.filename,
+                this.options.processedFilename
```

Comment:
What is this option? There is no documentation about it and it is not used by the test cases.